components/general/fonts/_fonts.scss

```diff
@@ -1 +1,16 @@
-// this file probably shouldn't be empty...
+.font {
```

Comment:
Hmm, so I suggest to delete this styles and just add files with fonts (in this case Futura and Bodoni) but the question is how are you adding fonts in webs? Can I use fontsquirrel for that or you have your own loading font? I have all from Vasyl in files.